templates/user/comment.html

```diff
@@ -2,6 +2,14 @@
 
 {% block js_media %}
     {% include "comments/base-media-js.html" %}
+    <script type="text/javascript">
+        $(function() {
+           $("#btn-delete-all").click(function(){
+              return confirm('{{ _('Are you sure you want to delete all comments?') }}');
+           });
+       });
+
+    </script>
```

Comment:
Please fix the indentation: lines 7-10 are missing a space

Also remove empty line 11